api_client/python/timesketch_api_client/sketch.py

```diff
@@ -1522,8 +1522,17 @@ def add_event(
             'message': message,
             'tag': tags
         }
-        if any(x in attributes for x in form_data):
-            raise ValueError('Attributes cannot overwrite values already set.')
+
+        duplicate_attributes = list()
+
+        for key in attributes.keys():
+            if key in form_data.keys():
+                duplicate_attributes.append(key)
```

Comment:
for this you could also just have done

```python
duplicate_attributes = [k for k in attributes if k in form_data]
```

that is do that in a single line using list comprehension